pkg/lang/golang.go

```diff
@@ -342,6 +352,11 @@ func GenerateGoTypesForPackage(s *schema.Schema, genConfig *config.GeneratorConf
 					yyy.PossibleTypes = append(yyy.PossibleTypes, ttt)
 				}
 
+				// Require at least one auto-generated method, allow others
+				yyy.Methods = make([]string, 2)
```

Comment:
I think this means that when the appends are started, that we will have two empty elements at the beginning of the slice.

Reply:
That is correct, the length of the empty slice would be `2`, then we would fill it without needing to reallocate.

This can be optimized to `len(interfaceMethods)+1` so there is no wasted memory/allocations.